kommunicate/src/main/java/io/kommunicate/KmConversationBuilder.java

```diff
@@ -199,6 +200,15 @@ public KmConversationBuilder setConversationTitle(String conversationTitle) {
         return this;
     }
 
+    public List<String> getUserIds() {
+        return userIds;
+    }
+
+    public KmConversationBuilder setUserIds(List<String> userIds) {
```

Comment:
@Sunilkumarr pls check if builder pattern uses method names as setX or withX?

Reply:
Mostly I see with setX in android this are the links for  example 
1.  https://dzone.com/articles/the-builder-pattern-for-class-with-many-constructo
2. https://www.vogella.com/tutorials/DesignPatternBuilder/article.html